app/src/main/res/layout/example_2_activity.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2017 Vandolf Estrellado
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~        http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <!-- We can also embed the fragment in xml using the fragment tag. However, we create the
+            fragment in Java for this example. -->
+        <FrameLayout
+            android:id="@+id/fragment_a_container"
+            android:layout_width="match_parent"
+            android:layout_height="0dp"
```

Comment:
There is a bug with using `layout_height="0dp"` and `layout_weight="1"` like this starting in Android N (API level 24+). The last line of each `FrameLayout`s are cut-off until both have been re-measured.

The fix is to just use `layout_height="wrap_content"` as seen here: https://github.com/vestrel00/android-dagger-butterknife-mvp/commit/450813684f323ef70d72c852a7a6604f60ffa12c